test/process_test.cc

```diff
@@ -22,6 +23,37 @@ using namespace testing;
 namespace Nighthawk {
 namespace Client {
 
+constexpr absl::string_view kSinkName = "{name:\"nighthawk.dummy_stats_sink\"}";
+int kNumFlushes = 0;
+
+class DummyStatsSink : public Envoy::Stats::Sink {
```

Comment:
Maybe add a short comment describing purpose of `DummyStatsSink`?

Reply:
Done